server/models/events_persister.go

```diff
@@ -154,13 +154,13 @@ func (e *EventsPersister) PersistEvent(event *events.Event) error {
 	return nil
 }
 
-func (e *EventsPersister) getCountBySeverity(userID uuid.UUID, eventStatus events.EventStatus) ([]*CountBySeverityLevel, error) {
+func (e *EventsPersister) getCountBySeverity(userID, sysID uuid.UUID, eventStatus events.EventStatus) ([]*CountBySeverityLevel, error) {
 	if eventStatus == "" {
 		eventStatus = events.Unread
 	}
 
 	eventsBySeverity := []*CountBySeverityLevel{}
-	err := e.DB.Model(&events.Event{}).Select("severity, count(severity) as count").Where("status = ? and user_id = ?", eventStatus, userID).Group("severity").Find(&eventsBySeverity).Error
+	err := e.DB.Model(&events.Event{}).Select("severity, count(severity) as count").Where("status = ? AND (user_id = ? OR user_id = ?)", eventStatus, userID, sysID).Group("severity").Find(&eventsBySeverity).Error
```

Comment:
@jerensl why is or against user_id and user_id . i think it should be system_id .

Reply:
@aabidsofi19 I just wonder if maybe there is a bug I didn't know, that's why I followed the same approach as the previous PR https://github.com/meshery/meshery/pull/11065/files#diff-94ae26e2758ca9153f353b080834d204c2a8a920a98d271b66cf251efb54b13e

Reply:
I also made the change to the system_id in the latest commit and it works in my local as well